firefox_media_tests/playback/test_video_playback.py

```diff
@@ -41,38 +44,25 @@ def test_playback_starts(self):
                                         level='WARNING')
 
     def test_video_playback_partial(self):
-        self.run_playback(timeout=120)
+        self.run_playback(duration=60)
 
     def test_video_playback_full(self):
         self.run_playback()
 
-    def run_playback(self, timeout=0):
+    def run_playback(self, interval=1, duration=None,
```

Comment:
Yes. But when this is refactored, those tests will call run_playback(...., interval=10, ...)